python/ambassador_diag/diagd.py

```diff
@@ -1051,7 +1051,7 @@ def _load_ir(self, rqueue: queue.Queue, aconf: Config, fetcher: ResourceFetcher,
 
         if mappings:
             for mapping_name, mapping in mappings.items():
-                app.kubestatus.mark_live("Mapping", mapping_name, mapping.namespace)
+                app.kubestatus.mark_live("Mapping", mapping_name, mapping.get('namespace'))
```

Comment:
This should have a default -- probably `mapping.get('namespace', Config.ambassador_namespace)`?

Reply:
Fixed